modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/source/PollingSourceWrapperTestCase.java

```diff
@@ -103,7 +101,6 @@ public class PollingSourceWrapperTestCase {
   @Before
   public void setUp() throws Exception {
     when(componentLocationMock.getRootContainerName()).thenReturn(TEST_FLOW_NAME);
-    setComponentLocationMock();
```

Comment:
Not needed because of the [InjectMocks](https://github.com/mulesoft/mule/pull/12465/files#diff-865978990efd2ce2ca126489962d054c8b98dac5ae4377f79ae6d7e1b94a0cbdR97) annotation which will introspect the instance's fields and set any that is compatible with any mock in this test class. Confirmed by debugging and actually running the tests (there would be an NPE if the componentLocation is null on the wrapper when calling onStart).

This is done by Mockito using reflection, it currently works even on JDK 17. I think even when mule-module-extensions-support is modularized this will continue working because the setAccessible will come from within the same module (even though it is not the same package).